### ISSUE TITLE
cc-logs-application-view: make the date range selector accessible

### ISSUE THREAD

**Issue by pdesoyres-cc** [CleverCloud/clever-components#1015]
The actual implementation of the date range selector, using a drop-down quick selector, is not accessible.

Statement from @florian-sanders-cc :
> We should replace the `cc-popover` with a component that implements the [ARIA Combobox Design Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/).
This is no easy task so I suggest we do like we have done for the `cc-pricing-header` component and rely on the [Shoelace Dropdown component](https://shoelace.style/components/dropdown).
>
> Last time I checked, it was using the proper ARIA attributes + the right keyboard shortcuts. I cannot assert that it's 100% compliant (I would need time to test it and study its code) but it's the best option we have right now.
> 
> In the future, we may want to produce our own but I don't think we should go there right now. This is not up to me alone to decide so we can discuss it in team meeting?
